Controller/Process/Result.php

```diff
@@ -214,13 +214,18 @@ protected function _validateUpdateOrder($order, $response)
     {
         $result = false;
 
-        $this->_adyenLogger->addAdyenResult('Updating the order');
-
```

Comment:
don't we want to keep this log so we now the order updating process is started ?

Reply:
I put it after the check for the result query param so it won't make any confusion that we are updating the order while we have already jumped out of it.

https://github.com/Adyen/adyen-magento2/pull/453/files#diff-bd7cef8f0429f77c78243ef5e37f68a5R228